GUI/MainInstall.cs

```diff
@@ -390,6 +390,19 @@ private void PostInstallMods(object sender, RunWorkerCompletedEventArgs e)
                 RetryCurrentActionButton.Visible = false;
                 UpdateChangesDialog(null, installWorker);
             }
+            else if(!result.Key && installCanceled){
+                // User cancelled the installation
+                AddStatusMessage("Manually cancelled installation process!");
+                SetDescription("The user cancelled the installation manually!");
+                UpdateChangesDialog(result.Value, installWorker);
+                RetryCurrentActionButton.Visible = true;
+                Util.Invoke(statusStrip1, () =>
+                {
+                    StatusProgress.Value = 0;
```

Comment:
It would be nice if we didn't have to directly manipulate specific properties of GUI objects here. Some sort of helper function to encapsulate this at a higher level.

`MainWait.cs` already has a partial API along those lines with `ShowWaitDialog` and `HideWaitDialog`. Maybe it should also have `FailWaitDialog`, which could then be called by the cancel and error blocks here? Looks like the parameters should be the messages to display.

Reply:
`HideWaitDialog` already gets passed a bool `success`, but it's not used yet.
Either we keep this and differentiate `success` or we create an explicit function.

Reply:
I thought the idea was to avoid hiding it, though, so the user can look at the errors.

Reply:
Well, as you mention it...
It would be a bit odd to have a funtion called HideWaitDialog to not hide the WaitDialog in case of an error...